app/src/androidTest/java/io/github/cmput301f19t19/legendary_fiesta/MyMoodTest.java

```diff
@@ -63,4 +62,128 @@ public void MyMoodFilterClickTest() {
         onView(withText(endsWith("Sad"))).check(matches(isDisplayed()));
     }
 
+    /*
```

Comment:
`git` is a version control system, so you don't need to comment out the code blocks to preserve them. You can just safely remove them.

In case you need this piece of code later, you can always go back to the `git` timeline to find it.